susemanager-utils/susemanager-sls/src/grains/mgr_server.py

```diff
@@ -44,7 +44,7 @@ def server_grains():
         else:
             grains["has_report_db"] = False
         rhndef = _simple_parse_rhn_conf(RHNCONFDEF)
-        if rhndef.get("product_name", "uyuni") == "SUSE Manager":
+        if rhndef.get("product_name", "uyuni") == "SUSE Multi-Linux Manager":
```

Comment:
Maybe we need to check for both here, or keep the old name.
Reason: for MLM 5.1 this grain is not used anymore. We connect a peripheral server now via a different way.
Either we will remove this completely, or we use it to allow to connect a 5.0 Server to a MLM 5.1 Hub. In that case we would need to check for the old name.